core/src/components/badge/badge.ionic.scss

```diff
@@ -145,3 +145,36 @@
   width: globals.$ion-scale-1000;
   height: globals.$ion-scale-1000;
 }
+
+// Badge Empty
+// --------------------------------------------------
+
+:host(:empty) {
+  --padding-start: 0;
+  --padding-end: 0;
+}
+
+:host(:empty[vertical]) {
+  border: globals.$ion-border-size-025 globals.$ion-border-style-solid globals.$ion-bg-surface-inverse;
```

Comment:
I don't see this border on badge when it's a status/hint. This should be removed to match design. This is also shown in the Avatar designs.

Reply:
The border is present when the badge is used inside other components, thats why I added this only when the [vertical] attribute is being used, as it means that is not being used isolated. 

Its true that for now, on the scope of this PR, this wasn't probably needed but it will for all the following PR that add support for usage in other components (avatar, tab-bar, button). As this is a common style for badge, I thought it made sense to add it already. 

Let me know if I understood correctly or might be missing something :)

Reply:
Double-checked with UX team and you're right. In fact, the border was removed from all use-cases. 

Commited that change.

Also change the CSS position rule to work not only when its empty, but just when the vertical prop is being used (also checked with UX).